package.json

```diff
@@ -25,7 +25,6 @@
         "gatsby-plugin-google-tagmanager": "^4.14.0",
         "gatsby-plugin-image": "2.19.0",
         "gatsby-plugin-manifest": "^4.14.0",
-        "gatsby-plugin-page-progress": "^2.2.1",
```

Comment:
are u sure we don't need this?

Reply:
Isn't it the dependency of other plugins?

Reply:
To use this plugin in the project we need to add its options to `gatsby-config.js` to enable it and make it work.
At the moment we con't have any configuration for it in `gatsby-config.js`
I remember we used it for academy which is already removed.

Reply:
great :smile: